### PR TITLE
Get max latency value

### PR DESCRIPTION
**Pull request by BrennanConroy**
Didn't write the regex for wrk

### CODE REVIEW

**Comment by sebastienros** [comment]
All good, unless you also prefer to use `MaxLatency`.

Why didn't you write the regex?

**Comment by BrennanConroy** [comment]
I would prefer MaxLatency actually.

Because I don't like regex

**Comment by BrennanConroy** [comment]
Ok, tried to Regex

**Comment by BrennanConroy** [comment]
Ping @sebastienros

File: src/BenchmarksClient/Workers/WrkWorker.cs [hunk 1]
```diff
@@ -233,6 +233,9 @@ private static Process StartProcess(ClientJob job)
                 var p99Match = Regex.Match(job.Output, String.Format(LatencyPattern, "99%"));
                 job.Latency.Within99thPercentile = ReadLatency(p99Match);
 
+                var p100Match = Regex.Match(job.Output, @"\s+Latency\s+[\d\.]+\w+\s+[\d\.]+\w+\s+([\d\.]+)(\w+)");
```
Comment (sebastienros) [inline src/BenchmarksClient/Workers/WrkWorker.cs]: Did you try it?

Comment (BrennanConroy) [reply src/BenchmarksClient/Workers/WrkWorker.cs]: On an online regex visualizer with an example of wrk's output

Comment (sebastienros) [reply src/BenchmarksClient/Workers/WrkWorker.cs]: I also tested it, it works

**Pull request merged**